java/ingest/src/test/java/sleeper/ingest/job/IngestJobRunnerTest.java

```diff
@@ -75,17 +73,12 @@
 import static sleeper.configuration.properties.table.TableProperty.TABLE_NAME;
 
 @RunWith(Parameterized.class)
-public class IngestJobQueueConsumerIT {
-
+public class IngestJobRunnerTest {
```

Comment:
This is still an integration test. We could split out the parts that work with S3, but I think this is fine if we just replace Test with IT.